theano/sandbox/cuda/basic_ops.py

```diff
@@ -12,6 +12,7 @@
 from theano import tensor, scalar, config
 from theano.gradient import grad_undefined
 from theano.scalar import Scalar
+from theano.sandbox.cuda import dimshuffle as cuda_dimshuffle
```

Comment:
Don't do that. This cause import problem. Remove that line and when you want to use cuda_dimshuffle, just use:
cuda_ndarray.cuda_ndarray.dimshuffle(...) instead.